src/Plugin/EntityMetaRelation/ListPage.php

```diff
@@ -355,14 +374,14 @@ protected function getBundleOptions(string $selected_entity_type): array {
    *   The current entity meta wrapper.
    *
    * @return array
```

Comment:
We need to update the return type here to array|null. Although I don't understand why we change the type here, we could keep returning an empty array and then when building the configuration check if is empty.